### ISSUE TITLE
[CI] JvmErgonomicsTests.testG1GOptionsForSmallHeapWhenTuningSet

### ISSUE THREAD

**Issue by andreidan** [elastic/elasticsearch#63828]
**Build scan**: https://gradle-enterprise.elastic.co/s/bl4o3pv3wfpia

**Repro line**: 
```
./gradlew :distribution:tools:launchers:test
```

**Reproduces locally?**: Yes

**Applicable branches**: 7.x

**Failure history**:

**Failure excerpt**:
```
org.elasticsearch.tools.launchers.JvmErgonomicsTests > testG1GOptionsForSmallHeapWhenTuningSet FAILED
    java.lang.AssertionError: 
    Expected: every item is not a string starting with "-XX:G1HeapRegionSize="
         but: an item was "-XX:G1HeapRegionSize=4m"
        at __randomizedtesting.SeedInfo.seed([36B0CC74B1A24ED9:41F1664009F0DA1A]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.tools.launchers.JvmErgonomicsTests.testG1GOptionsForSmallHeapWhenTuningSet(JvmErgonomicsTests.java:142)
```

**Comment by rjernst** [comment]
@ebadyano Can you take a look at this?

**Comment by ebadyano** [comment]
Closed via #63835
